### PR TITLE
RPRBLND-2229: Blender 3.4 integration

### PR DESCRIPTION
**Pull request by VascoPi**
### PURPOSE
Blender 3.4 integration. Implement Mix and Mix Color nodes.
image

### EFFECT OF CHANGE
Added support for shader nodes Mix and Mix Color.

### TECHNICAL STEPS
Added class ShaderNodeMix.
Moved repeated code used within `ShaderNodeMix` and `ShaderNodeMixRGB` to function `blend_color`.
Added pass poll argument to NodeItem to show a node for a particular blender version.

### NOTES FOR REVIEWERS
The changes affect the node's appearance in UI for different versions of Blender:
Mix, Mix RGB, Mix Color,
Combine XYZ, Combine RGB, Combine HSV, Combine Color,
Separate XYZ, Separate RGB, Separate HSV, Separate Color

### CODE REVIEW

File: src/rprblender/nodes/__init__.py [hunk 1]
```diff
@@ -103,7 +105,8 @@ def get_current_convert_nodes():
         NodeItem('ShaderNodeBrightContrast'),
         NodeItem('ShaderNodeGamma'),
         NodeItem('ShaderNodeInvert'),
-        NodeItem('ShaderNodeMixRGB'),
+        RPR_NodeItem("ShaderNodeMix", label="Mix Color", settings={"data_type": "'RGBA'"}, blender_version=">=3.4"),
```
Comment (bnagirniak) [inline src/rprblender/nodes/__init__.py:108]: We can change it to:
`NodeItem("ShaderNodeMix", label="Mix Color", settings={"data_type": "'RGBA'"}, poll=lambda cls: BLENDER_VERSION >= "3.4")`

**Review by bnagirniak — CHANGES_REQUESTED** [review]
Improvements in NodeItem

File: src/rprblender/nodes/__init__.py (hunk 1 above)
Comment (VascoPi) [reply src/rprblender/nodes/__init__.py]: Thanks, improved within https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon/pull/582/commits/5a251cc6f55f589967f41bfc55ab22951c98bdaa

**Review by bnagirniak — APPROVED** [review]
Looks good

**Review by DagerD — APPROVED** [review]

**Review by ARozghon — APPROVED** [review]
approved by QA

**Pull request merged**